### DISCUSSION TITLE
Setting TargetFrameworks to net8.0-browser is not currently supported

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/Uno.Wasm.Bootstrap#857] [Q&A] [answered]
I cannot start the wasm app if I set `<TargetFrameworks>net8.0-browser</TargetFrameworks>` in wasm project.

It seems like `Uno.Wasm.Bootstrap` is always using the .net8.0 directory.

**Comment by jeromelaban** [comment, marked as answer]
`net8.0-browser` is not a supported target framework (and never was). You need to use `net8.0-browserwasm` as defined in our templates.

**Reply by Daoting** [reply]
Thanks.
